go.mod

```diff
@@ -12,6 +12,7 @@ require (
 	github.com/spf13/cobra v1.8.1
 	github.com/stretchr/testify v1.9.0
 	golang.org/x/sync v0.8.0
+	gopkg.in/yaml.v3 v3.0.1
```

Comment:
you've used the wrong yaml library ;) we're using https://github.com/goccy/go-yaml